packages/preferences/src/browser/preferences-frontend-application-contribution.ts

```diff
@@ -30,13 +30,16 @@ export class PreferencesFrontendApplicationContribution implements FrontendAppli
 
     onStart() {
         this.schemaProvider.ready.then(async () => {
```

Comment:
no need to use `async`

Reply:
this line was not changed and not related to the issue